Makefile.in

```diff
@@ -228,6 +236,10 @@ config_files: ocaml-stage1-config.status
 	/bin/echo -n "( `grep \"^SHAREDLIB_CFLAGS=\" ocaml/Makefile.config \
 		| sed 's/^SHAREDLIB_CFLAGS=//'` )" > sharedlib_cflags.sexp
 
+.PHONY: ocaml_compiler_internal_param
+ocaml_compiler_internal_param: tools/ocaml_compiler_internal_params
+	cp tools/ocaml_compiler_internal_params ocaml/stdlib/
```

Comment:
I now see why the gitignore change below is needed, but I'd really like to avoid modifying the `ocaml/` directory during the build if possible.  (@lukemaurer has already done some work towards this.)  Can this file be found anywhere else?

Reply:
I can rewrote the Makefile to copy this file to the `@stageX_prefix@/lib directly, without ever going through src or build, would it be better?